sirepo/sim_data/__init__.py

```diff
@@ -194,6 +199,16 @@ def compute_model(cls, model_or_data):
     def delete_sim_file(cls, sim_id, basename):
         return cls._delete_sim_db_file(cls._sim_file_uri(sim_id, basename))
 
+    @classmethod
+    def example_paths(cls):
+        import sirepo.simulation_db
+        return sirepo.resource.glob_paths(
+            _TEMPLATE_RESOURCE_DIR,
+            cls.sim_type(),
+            cls._EXAMPLE_RESOURCE_DIR,
+            f'*{sirepo.simulation_db.JSON_SUFFIX}',
```

Comment:
This is an example where JSON_SUFFIX should be split off from simulation_db perhaps in sirepo.const or even elsewhere (pkconst?).

Reply:
53f4c531e